demo/src/main/AndroidManifest.xml

```diff
@@ -43,7 +43,42 @@
     <activity android:name="com.google.android.exoplayer.demo.PlayerActivity"
         android:configChanges="keyboardHidden|orientation|screenSize"
         android:label="@string/application_name"
-        android:theme="@style/PlayerTheme"/>
+        android:theme="@style/PlayerTheme">
+
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW"/>
+
+            <category android:name="android.intent.category.DEFAULT"/>
+            <category android:name="android.intent.category.BROWSABLE"/>
+
+            <data android:mimeType="video/*"/>
```

Comment:
- Why not also audio/*
- Should this be restricted to schemes http/https/file/asset/content?